### PR TITLE
Seed update

### PR DESCRIPTION
**Pull request by fayeed**

### CODE REVIEW

**Comment by fayeed** [comment]
@jdeepee PR is ready.

File: scripts/get-builtin-test-lang.js [hunk 1]
```diff
@@ -30,13 +30,19 @@ const languages = {
   },
   "direct-message-language": {
     bundle: "https://github.com/perspect3vism/direct-message-language/releases/download/0.0.4/bundle.js"
+  },
+  "direct-message-language": {
+    bundle: "https://github.com/perspect3vism/direct-message-language/releases/download/0.0.4/bundle.js"
+  },
```
Comment (jdeepee) [inline scripts/get-builtin-test-lang.js]: duplicate here

File: src/cli.ts [hunk 2]
```diff
@@ -145,22 +74,29 @@ export function startServer(relativePath: string, bundle: string, meta: string,
     fs.removeSync(path.join(process.cwd(), './src/test-temp'))
     fs.mkdirSync(path.join(process.cwd(), './src/test-temp'))
     fs.mkdirSync(path.join(process.cwd(), './src/test-temp/languages'))
```
Comment (jdeepee) [inline src/cli.ts]: lets use the `build/languages` directory here. In order for the languageLanguage to use this path you must pass `{"storagePath": "/complete/path/build/languages"}` into the `languageLanguageSettings` section in the bootstrap seed file.

**Review by jdeepee — COMMENTED** [review]
Looks great! Just the small changes mentioned here, will give it a try once these are resolved and ad4m-test is published!

**Pull request merged**